src/main/java/hudson/plugins/analysis/core/ResultAction.java

```diff
@@ -34,7 +34,7 @@
      *
      * @return the associated build of this action
      */
-    AbstractBuild<?, ?> getBuild();
+    Run<?, ?> getBuild();

```

Comment:
Not sure if this could be a problem. Presumably most callers are referring to some subtype, not this interface per se.